lib/asciidoctor/iso/term_lookup_cleanup.rb

```diff
@@ -28,22 +28,31 @@ def set_termxref_tags_target
         xmldoc.xpath('//termxref').each do |node|
           target = normalize_ref_id(node.text)
           if termlookup[target].nil?
-            log.add('AsciiDoc Input', node, "#{target} does not refer to a real term")
+            remove_missing_ref(node, target)
             next
           end
 
           modify_ref_node(node, target)
         end
       end
 
+      def remove_missing_ref(node, target)
+        warn(%Q(Error: Term reference in `term[#{target}]` missing: \
+                "#{target}" is not defined in document.).gsub(/\s+/, ' '))
+        log.add('AsciiDoc Input', node, "#{target} does not refer to a real term")
```

Comment:
This... is odd: the log already outputs its contents both to STDERR and to the .err logfile (sorted in the latter instance), so there is no need for a separate warning, let along to a different error message. Feel free to change the error message in the log to "Term reference in `term[#{target}]` missing: #{target}" is not defined in document.

Reply:
@ronaldtse said that did not see any warning during iso-34000 build, so i add explicit one.

Reply:
@w00lf I take it back -- I can see the warning now.

```
Error: Term reference in `term[representation]` missing: "representation" is not defined in document.
AsciiDoc Input: (XML Line 000568): representation does not refer to a real term
Error: Term reference in `term[expression]` missing: "expression" is not defined in document.
AsciiDoc Input: (XML Line 000583): expression does not refer to a real term
Error: Term reference in `term[feature]` missing: "feature" is not defined in document.
AsciiDoc Input: (XML Line 000589): feature does not refer to a real term
```

Let's keep only one: 

```
        log.add('AsciiDoc Input', node, "Error: Term reference in `term[#{target}]` missing: \
                "#{target}" is not defined in document")
```

Reply:
@ronaldtse done, removed `warn`, changed log message